### DISCUSSION TITLE
Change environmental variables from admin panel

### DISCUSSION THREAD

**Discussion by arielzusya** [alexjustesen/speedtest-tracker#1874] [Ideas]
It would be great to be able to adjust environmental variables from the admin panel rather than having to make changes from the cli. Perhaps this could be the ability for an admin to change the variable and then trigger a reload of the container to make those variables live. For example, if the smtp settings change, it would be nice to be able to do that from the admin panel. It would also be nice to add, remove, or otherwise alter the list of server ids from the admin panel. Finally, it would be nice to be able to change the schedule from the admin panel.

Thanks for your great work!

**Comment by alexjustesen** [comment]
I've got a much better solution in mind for handling scheduled tests (multiple schedules) and managing notification channels in the UI vs. as environment variables. No timeline though.

**Reply by arielzusya** [reply]
Very cool! Looking forward to seeing what you do. Thanks again for your hard work!
